### PR TITLE
#50557 - Order summary Shipping updates

### LINKED ISSUE #50557: [Checkout Shipping Options] Order summary Shipping updates
This is a followup to https://github.com/woocommerce/woocommerce/issues/46948

```[tasklist]
- [ ] When no address has been provided and there are no shipping options, change the wording `No shipping options available` in the order summary section to `Enter address to calculate`, and display it in 80% of the font size of the `Shipping` label and normal, neither `<strong>` nor `<em>`. (See image 1 for expected result)
- [ ] When an address _has_ been provided, change the wording `No shipping options available` in the order summary section to `No available delivery option`, and display it in 80% of the font size of the `Shipping` label and normal, neither `<strong>` nor `<em>`. (See image 2 for expected result)
```

### Image 1
image

### Image 2
image

### PR DESCRIPTION
**Pull request by samueljseay**
### Changes proposed in this Pull Request:

Closes #50557

* When the address is incomplete we show "Enter address to calculate"
* When the address is provided (but no shipping option available) we show "No available delivery option"
* Font size is changed to 80% of normal and no more bold style, also removed bold style from subtotal as per the design provided in #50557

Before:

 before-no-shipping 
 
 After:

Address provided but no shipping option available:
after-no-delivery-option

No address provided:
after-no-address

### How to test the changes in this Pull Request:

The way I tested this was to:

1. Have 1 shipping option to Netherlands
2. Add products to cart and go to checkout, without an address supplied "Enter address to calculate" should show.
3. Now provide an address but for a country we don't ship to
4. See the copy and styling for "No available delivery option".

Also test these conditions in dark mode, by choosing a dark color palette in the theme editor and change the checkout block to dark mode.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
Changes to copy and styling of delivery summary in checkout block.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/footer-item/style.scss [hunk 1]
```diff
@@ -11,4 +11,8 @@
 	.wc-block-components-totals-footer-item-tax {
 		margin-bottom: 0;
 	}
+
+	.wc-block-components-totals-item__value {
+		font-weight: bold;
```
Comment (samueljseay) [inline plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/footer-item/style.scss:16]: This is to retain the font weight being bold within the footer.

**Comment by samueljseay** [comment]
Something is up with the changelog CI action, but otherwise I think this is ready for review.

**Review by opr — CHANGES_REQUESTED** [review]
Thanks Sam, this is looking pretty good! While testing with Subscriptions I noticed an issue which I raised here: https://github.com/woocommerce/woocommerce-subscriptions/issues/4728

I also noticed an issue in Storefront, the text is larger than the row title and other values, could we fix that please?



According to [the design](https://www.figma.com/design/c8VYjXqluE9vlVSE1RgJix/%5BPublic%5D-Cart-%26-Checkout-Blocks?node-id=463-6541&node-type=frame&t=fiQT9tjgNOYZZr9g-0), the text on the right should be slightly lighter than the title however in Tove, TT4, and Storefront (not an exhaustive list, they're just the ones I tested) it's not. Was this a decision made on purpose because of themes with different colour palettes?

Besides those issues, it's working well in in Firefox, Safari, and Edge.

**Comment by samueljseay** [comment]
> According to [the design](https://www.figma.com/design/c8VYjXqluE9vlVSE1RgJix/%5BPublic%5D-Cart-%26-Checkout-Blocks?node-id=463-6541&node-type=frame&t=fiQT9tjgNOYZZr9g-0), the text on the right should be slightly lighter than the title however in Tove, TT4, and Storefront (not an exhaustive list, they're just the ones I tested) it's not. Was this a decision made on purpose because of themes with different colour palettes?

@opr Yeah I opted not to add it because we don't have a good way to support a dark pallette out of the box. I do realize that this could be contentious, but I didn't like the change because this area already looks ok with a dark pallette ootb, and at least it's readable, but going to a dark pallette would immediately make this piece of text unreadable. 

The user can switch to dark mode setting to fix this, so I'm ok to introduce the lighter color if you agree, but also @senadir mentioned removing the one low-emphasis text color we have so I wondered, what should this color be set to?

**Comment by samueljseay** [comment]
> I also noticed an issue in Storefront, the text is larger than the row title and other values, could we fix that please?

@opr thanks for checking this. I approached the font size completely wrong. I have updated it to use the em based mixin, but note that we don't have an exactly `80%` size (0.8em of the basis), so I've set it to the `0.875` size, because `0.75` felt too small.

**Comment by samueljseay** [comment]
I've requested re-review, but please let me know how you're feeling about the color. I'm open to changing it, and it was really just my personal idea to make it not look worse with dark pallettes.

**Review by opr — APPROVED** [review]
Thanks for your explanations Sam, I am fine with using 0.875 and your reasoning behind the colour makes sense. Happy to approve, also tagging @nikkivias here to check she's OK with us taking the initiative on this. Nikki if you'd like us to change/update anything let us know and we can iterate on it in a follow up.

**Pull request merged**